packages/svelte/src/internal/client/runtime.js

```diff
@@ -653,12 +653,15 @@ function process_effects(effect, filter_flags, shallow, collected_effects) {
 	}
 
 	if (effects.length > 0) {
+		var i;
 		if ((filter_flags & EFFECT) !== 0) {
-			collected_effects.push(...effects);
+			for (i = 0; i < effects.length; i++) {
```

Comment:
Probably good to add a comment here so we don't Regress in the future thinking "we can just use push(...effects) here"

Reply:
I instead refactored the following loop into this one.

Reply:
I added a comment though.